adagrams/game.py

```diff
@@ -1,11 +1,94 @@
+import random
+import string
+
+LETTER_POOL = {
+    'A': 9,
+    'B': 2,
+    'C': 2,
+    'D': 4,
+    'E': 12,
+    'F': 2,
+    'G': 3,
+    'H': 2,
+    'I': 9,
+    'J': 1,
+    'K': 1,
+    'L': 4,
+    'M': 2,
+    'N': 6,
+    'O': 8,
+    'P': 2,
+    'Q': 1,
+    'R': 6,
+    'S': 4,
+    'T': 6,
+    'U': 4,
+    'V': 2,
+    'W': 2,
+    'X': 1,
+    'Y': 2,
+    'Z': 1
+}
+
+LETTER_SCORE = {
+    ("A", "E", "I", "O", "U", "L", "N", "R", "S", "T"): 1,
+    ("D", "G"): 2,
+    ("B", "C", "M", "P"): 3,
+    ("F", "H", "V", "W", "Y"): 4,
+    ("K",): 5,
+    ("J", "X"): 8,
+    ("Q", "Z"): 10,
+}
+
+
 def draw_letters():
-    pass
+    letters = []
+    while len(letters) < 10:
+        random_letter = random.choice(string.ascii_uppercase)
```

Comment:
instead of bringing another library, let's use what we already have! Maybe we can turn our `LETTER_POOL` into a list and randomly choose one of the keys:

```suggestion
        random_letter = random.choice(list(LETTER_POOL))
```